src/lib/LUA/tx_devLUA.cpp

```diff
@@ -40,7 +40,7 @@ static struct luaItem_selection luaAirRate = {
 static struct luaItem_selection luaTlmRate = {
     {"Telem Ratio", CRSF_TEXT_SELECTION},
     0, // value
-    "Off;1:128;1:64;1:32;1:16;1:8;1:4;1:2",
+    "Std;Off;1:128;1:64;1:32;1:16;1:8;1:4;1:2;Race",
```

Comment:
On this order too.. Should Std come after Off (which was set as default on first flash)?

Reply:
was wrong about `Off` becoming default on first flash. I have 1:128 set before flashing and it carried over to the PR flashing. 1:128 was index 1, and on this PR, that becomes Off.